azure-pipelines.yml

```diff
@@ -125,7 +125,7 @@ jobs:
       TENANT_ID: $(TENANT_ID)
       CI_KIND_CLUSTER: true
       AZURE_ENVIRONMENT: AzureStackCloud
-      AZURE_ENVIRONMENT_FILEPATH: /etc/kubernetes/secrets-store-csi-providers/custom_environment.json
+      AZURE_ENVIRONMENT_FILEPATH: /provider/custom_environment.json
```

Comment:
what populates `/provider/custom_environment.json`? 
I see it was created by https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/test/kind-config.yaml#L7

Reply:
`/provider` is the container path for `/etc/kubernetes/secrets-store-csi-driver-providers` host path. As the provider mounts that volume, `custom_environment.json` is available in the pod.

Reply:
Ok thanks! I see you updated the mount path: `mountPath: "/provider"`